fluentd/README.md

```diff
@@ -87,6 +88,24 @@ Setup Example:
 </match>
 ```
 
+Additional parameters can be used to change the endpoint used in order to go through a proxy:
+
+* `host`: Proxy endpoint when logs are not directly forwarded to Datadog (default value is `intake.logs.datadoghq.com`)
+* `port`: Proxy port when logs are not directly forwarded to Datadog (default value is `10516`)
+* `use_ssl`: If `true`, the Agent initializes a secure connection to Datadog. In clear, TCP otherwise (default value is `true`)
+
+This also can be used to send logs to **Datadog EU** by setting:
+
+```
+<match datadog.**>
+
+  ...
+  host 'tcp-intake.logs.datadoghq.com'
```

Comment:
If this is a Datadog **EU** example, should the host entry here point to `.eu` instead?

Reply:
fixed